src/Entity/CitizenProject.php

```diff
@@ -297,7 +350,7 @@ public static function createForAdherent(
         string $problemDescription,
         string $proposedSolution,
         string $requiredMeans,
-        Committee $committee = null,
+        ?array $committees = null,
```

Comment:
It would be better to use `array $committees = [],` if possible.

Reply:
✔️